package.json

```diff
@@ -20,7 +22,6 @@
     "eslint-config-react-app": "5.0.2",
     "eslint-loader": "^3.0.2",
     "eslint-plugin-cypress": "^2.7.0",
-    "eslint-plugin-flowtype": "^4.3.0",
```

Comment:
I'm getting errors on this one, sure it should be removed?

Reply:
Fairly sure. We're not use Flow, so I don't think we need the plugin. I'm leaving the Cypress plugin because I'm optimistic we'll use it in future.

Reply:
Can't replicate it now.